python/xgbserver/poetry.lock

```diff
@@ -1363,7 +1363,7 @@ referencing = ">=0.31.0"
 
 [[package]]
 name = "kserve"
-version = "0.12.1"
+version = "0.13.0rc0"
```

Comment:
This should be 0.13.0rc1

Reply:
@sivanantha321 any idea why CI did not complain ?

Reply:
#3707